jina/serve/networking.py

```diff
@@ -966,7 +969,7 @@ async def _handle_aiorpcerror(
         # requests usually gets cancelled when the server shuts down
         # retries for cancelled requests will hit another replica in K8s
         self._logger.debug(
-            f'GRPC call errored, getting error {error} for the {retry_i + 1}th time.'
+            f'GRPC call to {current_deployment} errored, getting error {error} for the {retry_i + 1}th time.'
```

Comment:
Printing the `error.trailing_metadata()` has the underlying http error. I created #5469  for better formatting the grpc error.

Reply:
anyway, this will not harm right? I believe we can do this before

Reply:
It's not harmful.